octoprint_automation_scripts/__init__.py

```diff
@@ -183,10 +225,15 @@ def readline(self, *args, **kwargs):
                     self._temp_resp_len = len(self.g._p.temp_readings)
                     resp = self.g._p.temp_readings[-1]
                 else:
-                    if self.so is None or not self.so.response_string:
-                        resp = 'Automation script is running'
-                    else:
-                        resp = self.so.response_string
+                    resp = '>>> Automation Script Running'
+                    if self.so is not None and self.so.script_status:
+                        resp += ': ' + self.so.script_status
+                        if self.so.script_status != self._old_script_status:
+                            self._old_script_status = self.so.script_status
+                            payload = {'id': self.active_script_id,
+                                       'title': self.script_titles[self.active_script_id],
+                                       'status': self.so.script_status}
+                            eventManager().fire(Events.AUTOMATION_SCRIPT_STATUS_CHANGED, payload)
```

Comment:
I don't want `readline()` to become hard to reason about.  Can we factor this new stuff out into a method or something?